wof/core_1_1.py

```diff
@@ -50,6 +50,7 @@ def config_from_file(self, file_name):
 
         self.network = config.network.lower()
         self.vocabulary = config.vocabulary.lower()
+        self.urlpath = config.urlpath.lower()
```

Comment:
Could the `lower()` enforcement cause a problem in servers / OS's that allow for case-sensitive url's (other than the domain name, which per HTTP rules can not be case sensitive)? Is there a reason to force it to lower case?